src/kakarot/accounts/contract/contract_account.cairo

```diff
@@ -88,3 +88,21 @@ func is_initialized{
 }() -> (is_initialized: felt) {
     return ContractAccount.is_initialized();
 }
+
+// @notice This function is used to read the nonce from storage
+// @return nonce: The current nonce of the contract account
+@view
+func get_nonce{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (nonce: felt) {
+    return ContractAccount.get_nonce();
+}
+
+// @notice This function increases the contract accounts nonce by 1
+// @return nonce: The new nonce of the contract account
+@external
+func increment_nonce{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (
+    nonce: felt
+) {
+    Ownable.assert_only_owner();
```

Comment:
while this could be here, see that currently for all the other methods the access control is in the lib, so plz do the same

Reply:
Gotcha, will move this to the lib